main.js

```diff
@@ -0,0 +1,15 @@
+const noPosts = document.querySelector('li')
+// ^---variablt sets to select into the the first element w/li label
+
+noPosts.remove()
+// ^--.remove() the variable to remove desire section
+
+const noSpan = document.querySelector('.featured span')
+noSpan.remove()
+
+
+const noLastPost = document.querySelector('main .posts article')
+noLastPost.remove()
+
+const noPostAtAll = document.querySelector('main .posts h3')
```

Comment:
`querySelector` is only going to grab the first instance of `main .post h3`, therefore, in order to grab all of them you must use `querySelectorAll`